projects/packages/publicize/src/jetpack-social-settings/class-settings.php

```diff
@@ -172,18 +138,15 @@ public function get_settings( $with_available = false ) {
 		$this->migrate_old_option();
```

Comment:
Let's get rid of the call to this function

Reply:
We have to leave it there for SIG related migrations.

Reply:
Do we still need to migrate the SIG options? cc: @gmjuhasz

Reply:
We added that, because we changed the option structure before that change. If we get rid of the function, it could cause issues to people who still used the old version, and haven't used this version since to migrate. I think it's safer to keep it, but we could think over it

Reply:
Thanks for confirming!